python/cudf/cudf/tests/test_reshape.py

```diff
@@ -290,7 +290,9 @@ def test_df_merge_sorted_index(nparts, index, ascending):
     )
 
     expect = df.sort_index(ascending=ascending)
-    result = cudf.merge_sorted(dfs, by_index=True, ascending=ascending)
+    result = cudf.core.reshape._merge_sorted(
+        dfs, by_index=True, ascending=ascending
+    )
```

Comment:
Could also leave one of these as `cudf.merge_sorted` and put it in a `with pytest.warns:` block to check that the old path "works" with the expected message.

Reply:
This should also resolve the codecov failure

Reply:
Picked one and added the warning catch.